GEOSwiftTests/GEOS/SimplicityTestableTests.swift

```diff
@@ -0,0 +1,34 @@
+import XCTest
+import GEOSwift
+
+final class SimplicityTestableTests: XCTestCase {
+    let simplicityTestables: [SimplicityTestable] = [
+        Point.testValue1,
+        MultiPoint.testValue,
+        LineString.testValue1,
+        MultiLineString.testValue,
+        Polygon.LinearRing.testValueHole1,
+        Polygon.testValueWithHole,
+        MultiPolygon.testValue]
+
+    func testIsSimple() {
+        // multi point is simple iff it contains no repeated points
+        var multiPoint = MultiPoint(points: [.testValue1])
+
+        XCTAssertTrue(try multiPoint.isSimple())
+
+        multiPoint.points += multiPoint.points
+
+        XCTAssertFalse(try multiPoint.isSimple())
+    }
+
+    func testIsSimpleAllTypes() {
+        for g in simplicityTestables {
```

Comment:
Does g stand for geometry?

Reply:
Yep. In similar tests, this reads `for g in geometryConvertibles {`

I should probably make it `s` for these tests.